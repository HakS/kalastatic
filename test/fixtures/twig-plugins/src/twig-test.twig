---
title: Hello World!
---
<div class="twig-test">
  {{title|slug}}
</div>
