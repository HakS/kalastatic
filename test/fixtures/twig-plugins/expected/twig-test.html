<div class="twig-test">
  Hello-World!
</div>
